lib/msal-common/src/client/AuthorizationCodeClient.ts

```diff
@@ -248,10 +250,19 @@ export class AuthorizationCodeClient extends BaseClient {
         // Add sid or loginHint with preference for sid -> loginHint -> username of AccountInfo object
         if (request.sid) {
             parameterBuilder.addSid(request.sid);
+        } else if (request.account) {
+            const accountSid = this.extractAccountSid(request.account);
+            // If account and loginHint are provided, we will check account first for sid before adding loginHint
+            if (accountSid) {
+                parameterBuilder.addSid(accountSid);
+            } else if (request.loginHint) {
```

Comment:
Is this needed, given the same check is done below (265)?

Reply:
It will be skipped if account and loginHint are both passed in the request, which is why I included it here. Open to suggestions for cleaning this up.

Reply:
Ah right, makes sense.